ArtU/ArtU/Views/LessonView.swift

```diff
@@ -6,40 +6,51 @@
 //
 
 // Mock Data
-var mockLessons = ["Lesson 1", "Lesson 2", "Lesson 3", "Lesson 4"]
+struct Lesson: Identifiable, Hashable {
+    var id = UUID()
```

Comment:
Variables that are meant to be constant should be let. I am guessing that id should never change. Thus perhaps you should use let for id

Reply:
The YouTube video I watched used `var` instead of `let`. Here is an article on why: https://developer.apple.com/forums/thread/651802.